WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -161,7 +161,7 @@ public static boolean addOrUpdateBlog(String blogName, String xmlRpcUrl, String
                     blog.setCapabilities(capabilities);
                     blogUpdated = true;
                 }
-                if (isAutomatedTransfer) {
+                if (isAutomatedTransfer != blog.getAutomatedTransfer()) {
```

Comment:
Good change, this should future-proof this a tiny bit if there's ever a case where we can switch a site back from AT in the future.